vslib/sai_vs_nexthopgroup.cpp

```diff
@@ -13,4 +14,5 @@ const sai_next_hop_group_api_t vs_next_hop_group_api = {
 
     vs_bulk_create_next_hop_group_members,
     vs_bulk_remove_next_hop_group_members,
+    VS_GENERIC_QUAD_API(next_hop_group_map)
```

Comment:
add also unittests for those nae apis in unittest/vslin/test_sai_vs_nexthopgroup.cpp

Reply:
Done